firebase.json

```diff
@@ -75,6 +75,21 @@
       }
     ],
     "redirects": [
+      {
+        "source": "/build/developer-reference/intro-rollups",
+        "destination": "/build/developer-reference/protocol",
+        "type": 301
+      },
+      {
+        "source": "/build/developer-reference/fee-model",
```

Comment:
The `source` and `destination` look to be the same, is this expected?

Reply:
whoops, yeah don't know what happened there. Should be able to remove this without issue.